_data/projects/soundrecorder.yml

```diff
@@ -0,0 +1,15 @@
+name: SoundRecorder
+desc: A simple sound recording Android app implementing Material Design. The goal of this project is to develop as stable as possible sound recorder application.
+site: https://github.com/naXa777/SoundRecorder
+tags:
+- Java
+- Gradle
+- Android
+- mobile
+- sound-record
+- audio-recorder
+- dictaphone
+- material-design
+upforgrabs:
+  name: help wanted
+  link: https://github.com/naXa777/SoundRecorder/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@naXa777 Please change this to https://github.com/naXa777/SoundRecorder/labels/help%20wanted

Reply:
Thank you! Changed.